assembly/lib/uastc/common.ts

```diff
@@ -115,12 +115,12 @@ export function unpackTrits(t: i32): u32 {
  * Precompute unquantized 0..11 range: 1 trit, 2 bits
  */
 function storeUnq11(): void {
-  for (let bits = 0; bits < 4; bits++) {
-    const a = bits & 1 ? 0x1FF : 0;
-    const b = bits & 2 ? 0x116 : 0;
-    for (let trit = 0; trit < 3; trit++) {
+  for (let bits: u32 = 0; bits < 4; bits++) {
+    const a = (bits & 1) ? 0x1FF : 0;
+    const b = (bits & 2) ? 0x116 : 0;
+    for (let trit: u32 = 0; trit < 3; trit++) {
       const i = (trit << 2) | bits;
-      const u = (a & 0x80) | (((trit * 93 + b) ^ a) >>> 2);
+      const u = (a & 0x80) | (((trit * 93 + b) ^ a) >> 2);
       store<u8>(i, u, unq11Offset);
     }
   }
```

Comment:
Just wondering, what if unroll this two loop and merge 12 `store<u8>` into 3 `store<u32>` after constant folding and regrouping?
1) step 1. Unroll and constant folding:
```ts
// 0
store<u8>(0, 0,  unq11Offset);
store<u8>(4, 23, unq11Offset);
store<u8>(8, 46, unq11Offset);

// 1
store<u8>(1, 255, unq11Offset);
store<u8>(5, 232, unq11Offset);
store<u8>(9, 209, unq11Offset);

// 2
store<u8>(2, 69, unq11Offset);
store<u8>(6, 82, unq11Offset);
store<u8>(10, 107, unq11Offset);

// 3
store<u8>(3, 186, unq11Offset);
store<u8>(7, 163, unq11Offset);
store<u8>(11, 139, unq11Offset);
```
2) step 2 and final. Rearrange and group by 4 into `3` store operation:
```ts
store<u32>(0, (186 << 24) | (69 << 16) | (255 << 8) | (0 << 0), unq11Offset);
store<u32>(4, (163 << 24) | (82 << 16) | (232 << 8) | (23 << 0), unq11Offset);
store<u32>(8, (139 << 24) | (107 << 16) | (209 << 8) | (46 << 0), unq11Offset);
```

Reply:
I know it just call once but it could reduce size of code a little

Reply:
I'll open a new issue to track this as there are several more cases where precomputed values could be stored more efficiently.

JSYK, I saw that even minor changes in inlined initialization code may greatly affect generated WAT as the compiler may initially allocate more local variables.

Reply:
Binaryen and AS currently doen't unroll fixed size loops (especially nested loops) unfortunately. And I'm not sure it will be not improved in near future. So better do this manually if it make sense in some situations. 

Note: Unrolling unknown sized loops usually unnecessary due to VMs already perfrom this during JIT compilation and it's don't improve speed and just enlarge size of code.